src/storage/mod.rs

```diff
@@ -2448,6 +2498,34 @@ pub mod test_util {
     }
 }
 
+/// All statistics related to KvGet/KvBatchGet.
+#[derive(Debug, Default, Clone)]
+pub struct KvGetStatistics {
+    pub stats: Statistics,
+    pub perf_stats: PerfStatisticsDelta,
+    pub latency_stats: KvGetLatencyStats,
+}
+
+/// Latency indicators related to KvGet/KvBatchGet.
+///
+/// The detailed meaning of the indicators is as follows:
+///
+/// ```text
+/// ------> Begin ------> Scheduled ------> SnapshotReceived ------> Finished ------>
```

Comment:
I think this model is actually useful for all kv requests and even coprocessor requests. Is it possible to use this for all?

Reply:
If so, I think we can change it to a more general name.
cc @zhongzc

Reply:
Agreed

Reply:
In order to be used by multiple modules in the future, I moved it to [`components/tikv_kv/src/stats.rs`](https://github.com/tikv/tikv/pull/11508/files#diff-c206d7590a3c260a9ce287f7ee097981b45728491d48b4072e36fbe21633a679R268) and renamed it to `StageLatencyStats` which means `latency indicators for multi-execution-stages.`